src/app/screens/reset-password-screen/reset-password-screen.component.ts

```diff
@@ -0,0 +1,109 @@
+import { Component, OnDestroy, OnInit } from '@angular/core';
+import { FormBuilder, FormControl, FormGroup, Validators } from '@angular/forms';
+import { ActivatedRoute } from '@angular/router';
+import { Subject } from 'rxjs';
+import { takeUntil } from 'rxjs/operators';
+import { FeedbackInterface, FEEDBACK_STATUS } from 'src/app/components/feedback/feedback.interface';
+import { PasswordFormatValidator } from 'src/app/form-tools/validators/password-format.validator';
+import { ICON_STATUS, THEME } from 'src/app/material/button/button.component';
+import { CompanyService } from 'src/app/services/company/company.service';
+
+@Component({
+    selector: 'app-reset-password-screen',
+    templateUrl: './reset-password-screen.component.html',
+    styleUrls: ['./reset-password-screen.component.scss']
+})
+export class ResetPasswordViewComponent implements OnInit, OnDestroy {
+    public form: FormGroup;
+
+    public submitted = false;
+    public submitting = false;
+
+    public requestSucceeded: boolean = false;
+    public requestErrorMessage: string;
+
+    public buttonTheme = THEME;
+    public buttonStatus: string = ICON_STATUS.LOADING;
+
+    private _subscriptions = new Subject();
+    private _token: string;
+
+    public successFeedback: FeedbackInterface = {
+        status: FEEDBACK_STATUS.SUCCESS,
+        title: undefined,
+        subTitle: 'Nova password atualizada',
+        text: undefined,
+        actionLabel: 'Ir para Login',
+        url: ''
+    };
+
+    constructor(
+        private fb: FormBuilder,
+        private companiesService: CompanyService,
+        private activatedRoute: ActivatedRoute
+    ) {
+        this.form = this.fb.group({
+            password: [null, Validators.compose([Validators.required, PasswordFormatValidator])],
+            confirmPassword: [null, Validators.compose([Validators.required])]
+        });
+
+        this.successFeedback.url = this.activatedRoute.snapshot.queryParams.returnUrl || '/login';
```

Comment:
Neste caso, nunca deverá ter `returnUrl`

Reply:
não preencho nada simplesmente? não percebi bem.

Reply:
sorry 😛
como no caso anterior:
`this.successFeedback.url = '/login'`
`this.successFeedback.url = '/'`